homeassistant/components/tplink_lte.py

```diff
@@ -0,0 +1,157 @@
+"""
+Support for TP-Link LTE modems.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/tplink_lte/
+"""
+import asyncio
+import logging
+
+import aiohttp
+import attr
+import voluptuous as vol
+
+from homeassistant.components.notify import ATTR_TARGET, SERVICE_NOTIFY
+from homeassistant.const import (
+    CONF_HOST, CONF_NAME, CONF_PASSWORD, EVENT_HOMEASSISTANT_STOP)
+from homeassistant.core import callback
+from homeassistant.helpers import config_validation as cv, discovery
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+REQUIREMENTS = ['tp-connected==0.0.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'tplink_lte'
+DATA_KEY = 'tplink_lte'
+
+DEFAULT_DISCOVERY = True
+
+_NOTIFY_SCHEMA = vol.All(vol.Schema({
+    vol.Optional(CONF_NAME): cv.string,
+    vol.Required(ATTR_TARGET): vol.All(cv.ensure_list, [cv.string]),
+}))
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.All(cv.ensure_list, [vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(SERVICE_NOTIFY):
```

Comment:
Define `CONF_NOTIFY` in this component. Config keys should be stored in constants called `CONF_*`.

Reply:
Do you mean just rename `_NOTIFY_SCHEMA` to `CONF_NOTIFY` or something else?

Reply:
Replace `SERVICE_NOTIFY` with a locally defined `CONF_NOTIFY`.